Fails to link disobedience on bullseye

disorder master (d9bb6bef535c72baa4a494e717a8ceec5e1a26f6) fails to link on Debian bullseye as follows:

```
/bin/bash ../libtool  --tag=CC   --mode=link gcc -std=gnu99 -Wall -W -Wpointer-arith    -Wwrite-strings -Wmissing-prototypes    -Wmissing-declarations -Wnested-externs -Werror -Wshadow -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -pthread -isystem /usr/include/gtk-2.0 -isystem /usr/lib/x86_64-linux-gnu/gtk-2.0/include -isystem /usr/include/pango-1.0 -isystem /usr/include/atk-1.0 -isystem /usr/include/gdk-pixbuf-2.0 -isystem /usr/include/pango-1.0 -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/fribidi -isystem /usr/include/cairo -isystem /usr/include/pixman-1 -isystem /usr/include/harfbuzz -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/uuid -isystem /usr/include/freetype2 -isystem /usr/include/libpng16 -g -O2 -ffile-prefix-map=/home/cjwatson/src/packages/disorder/git/disorder=. -fstack-protector-strong -Wformat -Werror=format-security  -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lharfbuzz -lfontconfig -lfreetype -Wl,-z,relro -o disobedience disobedience.o client.o queue.o recent.o added.o queue-generic.o queue-menu.o choose.o choose-menu.o choose-search.o popup.o misc.o control.o properties.o menu.o log.o progress.o login.o rtp.o help.o settings.o users.o lookup.o playlists.o multidrag.o autoscroll.o globals.o memgc.o ../lib/libdisorder.a -lpcre2-8 -lgc -lgcrypt -lasound   -lm -lpulse-simple -lpulse -lpulse
libtool: link: gcc -std=gnu99 -Wall -W -Wpointer-arith -Wwrite-strings -Wmissing-prototypes -Wmissing-declarations -Wnested-externs -Werror -Wshadow -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -pthread -isystem /usr/include/gtk-2.0 -isystem /usr/lib/x86_64-linux-gnu/gtk-2.0/include -isystem /usr/include/pango-1.0 -isystem /usr/include/atk-1.0 -isystem /usr/include/gdk-pixbuf-2.0 -isystem /usr/include/pango-1.0 -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/fribidi -isystem /usr/include/cairo -isystem /usr/include/pixman-1 -isystem /usr/include/harfbuzz -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/uuid -isystem /usr/include/freetype2 -isystem /usr/include/libpng16 -g -O2 -ffile-prefix-map=/home/cjwatson/src/packages/disorder/git/disorder=. -fstack-protector-strong -Wformat -Werror=format-security -Wl,-z -Wl,relro -o disobedience disobedience.o client.o queue.o recent.o added.o queue-generic.o queue-menu.o choose.o choose-menu.o choose-search.o popup.o misc.o control.o properties.o menu.o log.o progress.o login.o rtp.o help.o settings.o users.o lookup.o playlists.o multidrag.o autoscroll.o globals.o memgc.o  -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lharfbuzz -lfontconfig -lfreetype ../lib/libdisorder.a -lpcre2-8 -lgc -lgcrypt -lasound -lm -lpulse-simple -lpulse -pthread
/usr/bin/ld: client.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: client.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: client.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: queue.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: queue.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: queue.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: recent.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: recent.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: recent.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: added.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: added.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: added.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: queue-generic.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: queue-generic.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: queue-generic.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: queue-menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: queue-menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: queue-menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: choose.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: choose.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: choose.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: choose-menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: choose-menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: choose-menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: choose-search.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: choose-search.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: choose-search.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: popup.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: popup.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: popup.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: misc.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: misc.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: misc.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: control.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: control.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: control.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: properties.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: properties.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: properties.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: menu.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: log.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: log.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: log.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: progress.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: progress.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: progress.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: login.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: login.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: login.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: rtp.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: rtp.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: rtp.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: help.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: help.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: help.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: settings.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: settings.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: settings.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: users.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: users.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: users.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: lookup.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: lookup.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: lookup.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: playlists.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: playlists.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: playlists.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
/usr/bin/ld: globals.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: multiple definition of `rtp_api'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:254: first defined here
/usr/bin/ld: globals.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: multiple definition of `login_window'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:231: first defined here
/usr/bin/ld: globals.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: multiple definition of `full_mode'; disobedience.o:./debian/build/disobedience/../../../disobedience/disobedience.h:170: first defined here
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:545: disobedience] Error 1
```

--- a/disobedience/disobedience.h
+++ b/disobedience/disobedience.h
@@ -167,7 +167,7 @@ void all_update(void);
 
 GtkWidget *menubar(GtkWidget *w);
 /* Create the menu bar */
-int full_mode;
+extern int full_mode;
 
 void users_set_sensitive(int sensitive);
 
@@ -228,7 +228,7 @@ extern const GtkTargetEntry choose_targets[];
 
 void login_box(void);
 
-GtkWidget *login_window;
+extern GtkWidget *login_window;
 
 /* User management */
 
@@ -251,7 +251,7 @@ void stop_rtp(void);
 void load_rtp_config(void);
 void save_rtp_config(void);
 void change_rtp_api(const char *api);
-const char *rtp_api;
+extern const char *rtp_api;
 int rtp_setvol(int *l, int *r);
 int rtp_getvol(int *l, int *r);
 

